docs/azure_arc_jumpstart/azure_arc_k8s/day2/arc_extensions/azure_monitor/index.md

```diff
@@ -0,0 +1,106 @@
+    ```
+
+* [Install or update Azure CLI to version 2.15.0 and above](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). Use the below command to check your current installed version.
+
+  ```shell
+  az --version
+  ```
+
+* Create Azure service principal (SP)
+
+    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcK8s" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcK8s",
+    "name": "http://AzureArcK8s",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+
+## Create Azure monitor extensions instance
+
+To create a new extension Instance, use `k8s-extension create`, passing in values for the mandatory parameters. The below example creates an instance of Azure Monitor extension on your Arc enabled Kubernetes cluster. An auotmation script file is provided part of jumpstart.
+
+* In order to keep your local environment clean and untouched, we will use [Google Cloud Shell](https://cloud.google.com/shell) to run the [*gke_monitor_onboarding*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/gke/gke_monitor/azure_monitor_k8s_extension.sh) shell script against the GKE connected cluster.
+
+* Before integrating the cluster with Azure Monitor for Containers, click on the "Insights (preview)" blade for the connected Arc cluster to show how the cluster is not currently being monitored.
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource](./01.png)
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource Insights](./02.png)
+
+* Edit the environment variables in the script to match your environment parameters and upload it to the Cloud Shell environment. After than run it using the ```. ./azure_monitor_k8s_extension.sh``` command.
+
+    > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+    ![Screenshot showing GKE cluster in GCP console](./03.png)
+
+    ![Screenshot showing connection to GKE cluster in GCP console](./04.png)
+
+    ![Screenshot showing uploading the script file to cloud shell](./07.png)
```

Comment:
img jump 04<>07

See [this section](https://azurearcjumpstart.io/scenario_guidelines/#screenshots) in the write-up guildeines.

Reply:
fixed and let me know if there any further updates needed. Thanks!